### DISCUSSION TITLE
AttributeError while loading 2D X-rays in 3D Slicer using MONAI Label

### DISCUSSION THREAD

**Discussion by neerajamaha** [Project-MONAI/MONAILabel#1105] [Q&A]
Hello,

I am using MONAI Label with the 3D Slicer viewer to segment some 2D x-ray images. I am running the multilabel radiology app with the deepgrow_2d model. However, I'd like to mention that I have not trained a model or run any inferences with a pre-trained model. I am using the MONAI Label workflow to manually segment a few hundred images with hopes of training and running predictions afterward. I have 2D DICOM files that I have converted to NIFTI for the workflow. I am running into an error when I attempt to fetch the next sample (pasted below), and thus I am not able to load those exams.
![image](https://user-images.githubusercontent.com/98112236/198708920-c59419ed-9d83-4fae-8e9b-e50c6be32de2.png)
This error only occurs for certain exams and not all. So I am able to load some exams, do my manual segmentations, and 'Submit Label'. I have tried to compare the headers, datatype, and IDs of the exams that do load versus those that don't, and do not see any differences. I have seen some discussions on this page of others who have used MONAI Label and 3D slicer for their 2D x-ray segmentation projects, so I was wondering if anyone has encountered this issue while loading their 2D data using the MONAI Label plugin? Any suggestions or guidance would be greatly appreciated.

Thank you!

**Comment by jlvahldiek** [comment]
Looks like you are using an outdated Slicer. Please get the latest stable release from here: https://slicer.org/download.html and also make sure to use the latest MONAILabel Slicer extension.

**Reply by neerajamaha** [reply]
Thank you for the suggestions. I have installed the latest version of Slicer and the MONAILabel extension, and the issue still persists.

![image](https://user-images.githubusercontent.com/98112236/198716025-58339754-e2fb-4cb9-9243-b021082141dd.png)
